luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/image_browser/mod.rs

```diff
@@ -53,44 +53,36 @@ impl ImageBrowser {
     pub fn update(&mut self, event: &dyn std::any::Any) {
         if let Some(event) = event.downcast_ref::<EditorEvent>() {
             match event {
```

Comment:
Dont spend your time to check this file, I just remove log and rustfmt update the format.